ssrc_config/config_indoor_sim.txt

```diff
@@ -0,0 +1,47 @@
+# Default parameter set for indoor flying in simulation (with regular gps simulation)
+# [ type: indoor_sim ]
+
+# Set waypoint acceptance radius to 0.5m
+param set NAV_ACC_RAD 0.5
+
+# Set height acceptance radius to 0.3m
+param set NAV_MC_ALT_RAD 0.3
+
+# RTL altitudes to avoid hitting the roof
+param set RTL_DESCEND_ALT 2
+param set RTL_RETURN_ALT 2
+
+# Offboard failsafe, set to land immediately even when RC is available
+param set COM_OBL_RC_ACT 4
+
+# RC failsafe, land immediately
+param set NAV_RCL_ACT 3
+
+# Cruise speed
+param set MPC_XY_CRUISE 0.5
+
+# Track trajectory more aggressively (default 0.5)
+param set MPC_XY_TRAJ_P 0.7
+
+# Increase velocity controller P gain
+param set MPC_XY_VEL_P_ACC 2.4
+
+# Make it accelerate faster upwards at takeoff
+param set MPC_TKO_RAMP_T 1.0
+
+# Limit upward movement speed for safety
+param set MPC_Z_VEL_MAX_UP 1.0
+
+# Smoothing trajectory a bit when using AutoLineSmoothVel
+param set MPC_JERK_AUTO 8
+param set MPC_ACC_HOR 3
+
+# Default Flight Modes and Kill Switch
+param set COM_FLTMODE1 0
+param set COM_FLTMODE4 2
+param set COM_FLTMODE6 11
+param set RC_MAP_FLTMODE 5
+param set RC_MAP_KILL_SW 7
+
+# Disable log encryption for indoor flights
+param set SDLOG_ALGORITHM 0
```

Comment:
It might be good to have logging enabled. I oftenly use it for flight review after a simulation test. Although, I am not sure if this parameter affects the log recording in simulation.

Reply:
As far as I know, this parameter affects to SITL also, so I can remove that config from sim case.

Reply:
Ah.. I misunderstood that param functionality. It is not disabling the logging, but only the encryption of it. Idea is that if flying indoor in safe location, there is no point to encrypt the PX4 flight data, but it can be stored in raw format.
So, in simulation the log-file is also in safe environment, so it does not need to be encrypted there either.

Reply:
This is okayish; I would still prefer keeping encryption on also in indoors flying. It may be nasty to leave the feature untested in indoor testing and only test is when going outdoors. It increases CPU load, also any problems there might render log files un-usable. So early error finding there is preferred.

Of course, it is a nuisance to always de-crypt the logfiles when uploading them. Maybe we could instead add some small script somewhere to do that.